test/test_transforms_job_run_task.py

```diff
@@ -47,7 +47,7 @@ def assert_docker_worker(task):
         "extra": {},
         "label": "fake-task-label",
         "routes": [],
-        "scopes": ["secrets:get:project/taskcluster/gecko/hgfingerprint"],
+        "scopes": [],
```

Comment:
I guess we can't remove the scope from ci-config until the Gecko patch rides the trains everywhere..

Reply:
Right.  I'm planning to let it ride 108, and uplift to ESR 102.6 or 102.7.